docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -65,3 +65,44 @@ After you save and exit the editor, `tctl` will update the resource:
 cluster networking configuration has been updated
 ```
 
+## Starting a database session
+
+Starting from version `17.1`, users can establish database sessions using the
+Teleport WebUI.
+
+To start a new session, locate your database in the resources list and click
+"Connect".
+
+![Resources list](../../img/connect-your-client/webui-db-sessions/resources-list.png)
+
+For supported databases, the dialog will present the option to start
+the session in your browser.
+
+![Resources list database connect dialog](../../img/connect-your-client/webui-db-sessions/resources-connect-dialog.png)
```

Comment:
Can you separate out the link here embedded in the `resources-connect-dialog.png` image ("Note: To connect with a GUI database client, see our documentation for instructions")? This make it look like folks can click it.

Reply:
Removed the link from the image.